policies/trademark-policy.md

```diff
@@ -1,139 +1,122 @@
 
-### Visual Trademark Instructions
+- Do not use CF Marks (Logos or Word Marks) in any commercial or marketing context other than as expressly permitted in this policy unless you have obtained explicit written permission from the Commonhaus Foundation to do so.
 
-To maintain the integrity of The Commonhaus Foundation's visual trademarks:
+- Do not use the Commonhaus Foundation trademarks more prominently than your own company, product or service name.
 
-- **Color:** Stick to official colors. Any deviations can weaken brand recognition.
-- **Size:** Maintain a minimum size ([specific dimensions]) to ensure legibility.
-- **Spacing:** Keep a clear boundary around the logo ([specific dimension]) clear of other elements.
-- **Alterations:** Preserve the original logo. Avoid distortions, rotations, and other modifications.
-- **Background:** Ensure that the logo is set against contrasting backgrounds for visibility.
+- Do not refer to a product or service as being certified under any CF Mark unless your company has explicit authorization to use such terms from the Commonhaus Foundation.
 
-## Commonhaus Trademarks and Crowdfunding
+- Do not attempt to claim or assert any ownership rights in any mark of the Commonhaus Foundation and do not attempt to register any Commonhaus Foundation trademark as a trademark, trade name, domain name, or “doing business as” name, alone or (unless specifically licensed) in combination with your own trademarks.
 
-Want to use a Commonhaus trademark for crowdfunding? Here's what you need to know and do.
+- Do not use a logo of the Commonhaus Foundation on posters, brochures, signs, websites, crowdfunding sites, or other marketing materials to promote your events, products or services without written permission from the Commonhaus Foundation.
 
-The Commonhaus Foundation recognizes that some projects may wish to use a project trademark when raising funds through crowdfunding platforms.
-These activities are not official foundation efforts, even if they are coordinated by core participants from our projects.
-Since crowdfunding operates externally to the Commonhaus Foundation, you must obtain permission before using a Commonhaus trademark on any crowdfunding page.
+### Use in Community Engagement
 
-In exchange for using the mark, the Commonhaus requires the administrator of the crowdfunding effort to adhere to these three policies:
+Marks may be used to discuss and reference CF projects provided the use does not imply CF endorsement and is consistent with the preservation of the goodwill and value of the Mark.
 
-1. You must have a policy that contributors and recipients disclose their legal or known names and company affiliations to the administrator of the funds.
+- Do not use the CF Marks in a manner that would disparage the Commonhaus Foundation or its projects (e.g., untruthful advertising, false/misleading promotional materials, etc.).
```

Comment:
Definitely NAL, but I'm surprised that this is compatible with the 1A. I mean, I understand that "untruthful advertising" and other commercial speech may be regulated, but it's not clear to me to what extent "Community Engagement" really qualifies as commercial, nor that our ownership of a TM has much bearing on that. 

Pretty sure people are allowed to go on Twitter and disparage Commonhaus and Hibernate to their heart's content and there's not a damn thing we can do about it.

Is this a common thing to find in TM policies?

Reply:
I linked my sources... Have more? =)

I can and will double check. The statement about untruthful materials is sound, but I'll ask with regard to other opinions. ;)

Reply:
I guess my issue is that this is in a section titled "Use in Community Engagement". Which doesn't sound like it's really directly about _commercial speech_.

Reply:
good point. Let me chew on it